shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -1053,6 +1053,18 @@ private void diposeAllViews() {
     }
   }
 
+  public void onTrimMemory(int level) {
```

Comment:
There are several levels of trim memory. Did you want to filter this to only a subset of memory calls? Also is it safe to call clearSurface multiple times because I think for a normal backgrounded app this will get called 2-4 times.

Reply:
I've filtered this to only run on trim level >= 40 (which is the level that triggers the bug).